app/controllers/api/health.php

```diff
@@ -920,8 +952,8 @@
         Event::AUDITS_QUEUE_NAME,
         Event::MAILS_QUEUE_NAME,
         Event::FUNCTIONS_QUEUE_NAME,
-        Event::USAGE_QUEUE_NAME,
-        Event::USAGE_DUMP_QUEUE_NAME,
+        Event::STATS_USAGE_QUEUE_NAME,
```

Comment:
we should also add the resource stats queue here?

Reply:
done